chef/cookbooks/docker/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Cookbook version
```

Comment:
Per overall review comment, this file should not be added directly to this repo.